openLRSng/RX.h

```diff
@@ -379,7 +379,7 @@ uint8_t bindReceive(uint32_t timeout)
       rfmGetPacket(rxc_buf, len);
 
       if (rxc_buf[0] == 'b') {
-        for (uint8_t i = 0; i < sizeof(bind_data); i++) {
+        for (uint8_t i = 0; i < sizeof(bind_data) && i < sizeof(rxc_buf); i++) {
```

Comment:
didn't catch this. compiler complains about possible overflow ?? bind_data is currently ~20bytes. oh well...

Reply:
The problem is it can't know the size of `bind_data` at compile time of the file, since it's external.